docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -0,0 +1,184 @@
+// GeometryOfRegionB        'progressives'
+// ...
+//
+// We can create a choropleth map by matching the winners of each region to one color by using buckets
+// This will create the following correspondence:
+//      'conservatives' <=> blue
+//      'progressives'  <=> red
+color: ramp(buckets($winner, ["Conservative Party", "Labour Party"]), [blue, red])
+```
+<div class="example-map">
+    <iframe
+        id="election-basic"
+        src="/developers/carto-vl/examples/maps/guides/ramp/election-basic.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+<div style="margin-bottom: 20px !important">
+   <a href="/developers/carto-vl/examples#example-election---basic">View my source code!</a>
+</div>
+##### *Others*
+
+When working with categories, the concept of the *others bucket* arises. For example, the buckets function picks some categories, but, what happens with the unselected categories?
+
+In the previous example, we could have regions in which the 'socialist' party won. This category wasn't placed in the `buckets` function so it will fall back to the `others` bucket.
+
+The `others` bucket will be colored gray by default. However, it's possible to override this behavior by providing a third parameter to `ramp`: `ramp(buckets($winner, ['conservatives', 'progressives'], [red, blue], white)`.
+
+<div class="example-map">
+    <iframe
+        id="election-others-bucket"
+        src="/developers/carto-vl/examples/maps/guides/ramp/election-others-bucket.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-election---others-bucket">View my source code!</a>
+</div>
+
+##### Showing the most common categories: `top`
+
+If we don't care about which colors get each category, but we don't want to color every category in the dataset, we can use `top` to group all uncommon categories in the *others bucket*.
+
+`top($cause, 5)` function will keep the five most common categories (regarding the entire dataset) and will group the rest into the *others bucket*.
+
+Let's see this with a dataset of US railroad accidents.
+
+<div class="example-map">
+    <iframe
+        id="accidents-top"
+        src="/developers/carto-vl/examples/maps/guides/ramp/accidents-top.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-railroad-accidents---top-expression">View my source code!</a>
+</div>
+
+##### Showing every category without selecting each color
+
+Sometimes, we don't care about the correspondence between colors and categories nor about having too many categories. This is particularly useful for getting quick feedback and exploring a dataset, but it is of reduced utility in later stages.
```

Comment:
We talked about trying to use the same voice "you don't care about" instead of "we don't care about" in the #950 guidelines

Reply:
Uff... that would be a huge change... Also, it may feel a little robotic to always use the same voice I feel

Reply:
Well, we can try with new guides, and improve these later on